src/intro-to-c/strings/strings-practice.md

```diff
@@ -0,0 +1,13 @@
+# Strings Practice
+
+1. Write a function that receives a word and return its length without using any libraries.
+
+2. Write a function that receives a word and turns it into uppercase.
+
+3. Write a function that receives two words and checks if they are equal.
+
+4. Write a function that receives a word and check if it is a palindrome.
+
+5. Write a function that receives a word and reverses it.
+
+6. **CHALLENGE** How would you read a first and last name and display it to the screen?
```

Comment:
Reword because I was super confused at first what this meant
```suggestion
6. **CHALLENGE:** How would you read a first and last name from the terminal, save it to a string, and display it to the screen?
```